test/test_torch.py

```diff
@@ -202,20 +202,20 @@ def normfn_attr(t, dim, keepdim=False):
             fn_attr = getattr(torch, fn_name) if fn_name != "norm" else normfn_attr
 
             def fn(t, dim, keepdim=False):
-                ans = fn_attr(x, dim, keepdim)
+                ans = fn_attr(x, dim, keepdim=keepdim)
                 return ans if not isinstance(ans, tuple) else ans[0]
 
             dim = random.randint(0, 2)
-            self.assertEqual(fn(x, dim).unsqueeze(dim), fn(x, dim, True))
+            self.assertEqual(fn(x, dim).unsqueeze(dim), fn(x, dim, keepdim=True))
             self.assertEqual(x.ndimension() - 1, fn(x, dim).ndimension())
-            self.assertEqual(x.ndimension(), fn(x, dim, True).ndimension())
+            self.assertEqual(x.ndimension(), fn(x, dim, keepdim=True).ndimension())
 
             # check 1-d behavior
             x = torch.randn(1)
             dim = 0
-            self.assertEqual(fn(x, dim), fn(x, dim, True))
+            self.assertEqual(fn(x, dim), fn(x, dim, keepdim=True))
             self.assertEqual(x.ndimension(), fn(x, dim).ndimension())
-            self.assertEqual(x.ndimension(), fn(x, dim, True).ndimension())
+            self.assertEqual(x.ndimension(), fn(x, dim, keepdim=True).ndimension())
```

Comment:
They weren't. I just kept commits separated before opening the PR, but the changes were in fact done in one sweep. Would you like me to squash the commits?